### PR TITLE
improve each plugin README.md and unify their format

### PR DESCRIPTION
**Pull request by nerd0geek1**
## Description
I improved each plugin's README.md content and unified its format.

What I did is 
- move `#h1` tag to at the top of README.md(`android_alarm_manager_plus`, `android_intent_plus`, `share_plus`)
- fix broken image link(`connectivity_plus`, `network_info_plus`, `package_info_plus`, `sensors_plus`, `share_plus`)
- insert/fix GitHub Actions badge(all plugin)

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

### CODE REVIEW

**Comment by vbuberen** [comment]
Thanks for your contribution. Would like to admit that some of readme files were already updated in other PRs:
#666, #667, #670 

I would suggest to revert your changes in readme for `share_plus`, `package_info_plus` and `sensors_plus`, since they are modified in those mentioned PRs anyway.

**Comment by nerd0geek1** [comment]
@vbuberen 
Thank you for reviewing my PR.

As you pointed out in https://github.com/fluttercommunity/plus_plugins/pull/673#issuecomment-1003554486, I deleted some commits from my PR.

Please review my PR again when you can.

File: packages/network_info_plus/network_info_plus/README.md [hunk 1]
```diff
@@ -3,10 +3,12 @@
 [![Flutter Community: network_info_plus](https://fluttercommunity.dev/_github/header/network_info_plus)](https://github.com/fluttercommunity/community)
 
 [![pub package](https://img.shields.io/pub/v/network_info_plus.svg)](https://pub.dev/packages/network_info_plus)
+[![network_info_plus](https://github.com/fluttercommunity/plus_plugins/actions/workflows/network_info_plus.yaml/badge.svg)](https://github.com/fluttercommunity/plus_plugins/actions/workflows/network_info_plus.yaml)
 
 <p class="center">
-<center><a href="https://flutter.dev/docs/development/packages-and-plugins/favorites" target="_blank" rel="noreferrer noopener"><img src="../../website/static/img/flutter-favorite-badge.png" width="100" alt="build"></a></center>
+<center><a href="https://flutter.dev/docs/development/packages-and-plugins/favorites" target="_blank" rel="noreferrer noopener"><img src="../../../website/static/img/flutter-favorite-badge.png" width="100" alt="build"></a></center>
```
Comment (nohli) [inline packages/network_info_plus/network_info_plus/README.md:9]: Is this link correct, or do we need another `../`

https://github.com/nerd0geek1/plus_plugins/blob/unify_readme_format/packages/package_info_plus/package_info_plus/README.md

**Review by nohli — APPROVED** [review]

**Comment by vbuberen** [comment]
Looks good. to me. Thanks for your contribution @nerd0geek1

**Review by vbuberen — APPROVED** [review]

**Comment by vbuberen** [comment]
@mhadaily Can I ask you to help us merge this PR? Seems like I can't enable additional required checks, so can't merge this PR.

**Comment by mhadaily** [comment]
Thank you both

**Comment by vbuberen** [comment]
I see a point to improve in current Actions setup - will open a few PRs soon to not run full checks for docs changes.

**Comment by mhadaily** [comment]
that would  be great @vbuberen

**Pull request merged**